### PR TITLE
Verify API Snippets

### PR DESCRIPTION
**Pull request by lornajane**
Adding the code snippets for the Verify API.

### CODE REVIEW

File: verify/check.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	verifyClient := vonage.NewVerifyClient(auth)
+
+	response, errResp, err := verifyClient.Check(os.Getenv("REQUEST_ID"), os.Getenv("CODE"))
+
+	if err != nil {
+		fmt.Printf("%#v\n", err)
+	} else if response.Status != "0" {
+		fmt.Println("Error status " + errResp.Status + ": " + errResp.ErrorText)
+	} else {
+		// all good
```
Comment (GregHolmes) [inline verify/check.go]: This comment feels like it sticks out a bit. I would guess it was there when writing the code?

**Review by GregHolmes — APPROVED** [review]
It's just the comment that sticks out I think. Everything looks ok to me!

**Comment by GregHolmes** [comment]
> It's just the comment that sticks out I think. Everything looks ok to me!

Shouldn't have approved there. I meant to leave a comment instead! Sorry!

File: verify/check.go (hunk 1 above)
Comment (lornajane) [reply verify/check.go]: Yes I think if we don't have other comments, this shouldn't be here either. Removed.

**Review by GregHolmes — APPROVED** [review]
Ran through all the examples and worked a charm! No issues with the code.

**Pull request merged**